### ISSUE TITLE
Search results not correctly filtering by collection date unless both start and end date are included

### ISSUE THREAD

**Issue by kyule** [BioKIC/NEON-Biorepository#359]
When I try to search for all samples collected after a certain date by putting in only a start date, I get no results: 

https://biorepo.neonscience.org/portal/collections/list.php?db=24&includeothercatnum=1&eventdate1=2022-02-01&usethes=1&taxontype=1

When I try to search for all samples collected before a certain date by putting in only an end date, I get all results: 

https://biorepo.neonscience.org/portal/collections/list.php?db=24&includeothercatnum=1&eventdate2=2019-02-01&usethes=1&taxontype=1

Only when I include both a start and end date I do get results that have correctly filtered by event dates. This appears to also be occurring in SEINet.

**Comment by egbot** [comment]
We should talk about this, since it's more of an issue with the interpretation of how the search form works, rather than a bug. Within other Symbiota portals the fields are not tagged as start and end dates. When only a single date is supplied, it looks for records that were collected on that date, which is what most folks would want. It only searches for a range when both dates are entered. However, it you only enter a end date, than it assumes you are looking for all records prior to that date.

**Comment by kyule** [comment]
I thought that might be the case.... I do think that the way I had interpreted how the range search would work is most intuitive (Andrew and I discussed and he concurs). I understand that changing it across the board may not be desirable, though. As a workaround, I updated the assistive text, since I do think that it is currently misleading by implying that a single date in the collection end date field would be meaningful. [With this ](https://github.com/BioKIC/NEON-Biorepository/pull/362)
